shell/platform/android/test/io/flutter/embedding/android/SplashShadowResources.java

```diff
@@ -9,9 +9,10 @@
 import org.robolectric.annotation.Implements;
 import org.robolectric.annotation.RealObject;
 import org.robolectric.shadow.api.Shadow;
+import org.robolectric.shadows.ShadowResources;
 
 @Implements(Resources.class)
-public class SplashShadowResources {
+public class SplashShadowResources extends ShadowResources {
```

Comment:
Robolectric expects to find an extension of `ShadowResources` when `@Implements(Resources.class)` is specified, as per [the docs]([the docs](https://robolectric.org/extending/)).